File: KevinRene/src/cons/ql/ast/expr/binary/Add.java
package cons.ql.ast.expr.binary;

import cons.ql.ast.Binary;
import cons.ql.ast.Expr;

public class Add extends Binary {

	final static String operator = "+";
	
	public Add(Expr left, Expr right) {
		super(left, right, operator);
	}
}

File: FelipezConde/QuestionnaireLanguage/AST/Nodes/Expressions/Binary/Add.cs
using AST.Nodes.Interfaces;

namespace AST.Nodes.Expressions.Binary
{
    public class Add : BaseBinary
    {
        public Add(BaseExpression left, BaseExpression right, PositionInText position)
            : base(left, right, position) 
        {}

        public override string ToString()
        {
            return "+";
        }

        public override T Accept<T>(ASTVisitors.Interfaces.IExpressionVisitor<T> visitor)
        {
            return visitor.Visit(this);
        }
    }
}
